source/includes/table-pre-6.0-compatibilty.rst

```diff
@@ -0,0 +1,24 @@
+.. list-table:: 
```

Comment:
[praise]

Thanks for this! This presentation is easily understood at a glance.